src/main/java/org/sciborgs1155/robot/commands/NoteVisualizer.java

```diff
@@ -171,20 +184,26 @@ private static Pose3d[] generatePath() {
             .unit()
             .times(shotVelocity)
             .plus(driveVelocity)
+            .plus(driveRotationVelocity)
             .times(-1);
 
     pathPosition = new ArrayList<>();
 
+    int step = 0;
     while (inField(pose) && pose.getZ() > 0) {
-      pathPosition.add(pose);
+      if (step % 3 == 0) {
```

Comment:
this stepping code is now duplcated between `shoot()` and `generatePath()`. remember that it is a good idea to have the note follow every point on the path, but a bad idea to actually send the entire path to nt at once.